mantidimaging/main.py

```diff
@@ -57,7 +66,13 @@ def parse_args() -> argparse.Namespace:
 def setup_application() -> QApplication:
     QGuiApplication.setAttribute(QtCore.Qt.AA_EnableHighDpiScaling)
     q_application = QApplication(sys.argv)
-    q_application.setStyle('Fusion')
+    if theme_selection:
+        q_application.setStyle(settings.value('theme_selection'))
+
+    default_font = QFont('-1')
```

Comment:
For me on linux I needed to change this to just `default_font = QFont()` to get the OS value. With the `-1` it will always give 12, Noto Sans.

```
QFontInfo(QFont('-1')).pointSize()=12
QFontInfo(QFont('-1')).family()='Noto Sans'
QFontInfo(QFont()).pointSize()=14
QFontInfo(QFont()).family()='Times New Roman'
```

Reply:
I'm happy for the check to only be on linux. One last thing to try is getting it from a widget.
```
QWidget().font().pointSize()
QWidget().font().family()
```